php/class-fieldmanager-zone-field.php

```diff
@@ -202,6 +214,23 @@ public function presave( $values, $current_values = array() ) {
 			}
 		}
 
+		/**
+		 * Alter values before they go through the sanitize & save routine.
+		 *
+		 * Here, we're enforcing $post_limit.
+		 *
+		 * @param  array $values Field values being saved.
+		 * @param  array $current_values Field's previous values.
+		 * @return array Altered values.
+		 */
+		public function presave_alter_values( $values, $current_values = array() ) {
+			// Ensure there's no more than $post_limit entries, if set
+			if ( is_array( $values ) && $this->post_limit ) {
```

Comment:
I wasn't concerned about it because this is developer-provided, but I added an "if > 0" check just to be sane.